src/screens/Customers/CustomerMain.tsx

```diff
@@ -83,23 +130,43 @@ function CustomerMain(props: RouteComponentProps & UserProps) {
     setFilteredCustomers(customers);
   }
 
+  const handleMenuSelect = (e: any) => {
```

Comment:
Kinda bad on my part because i let the `handleSearchChange` function merge with an `any` parameter type, but MaterialUI types these the following way (i took this typing from one of their sample code), so let's try to follow suit?

![image](https://user-images.githubusercontent.com/35501399/108914077-b7fe8680-75df-11eb-9fc1-e842c4476a29.png)